rosidl_adapter/test/data/action/Test.action

```diff
@@ -15,7 +20,11 @@ uint64 uint64_value
 string string_value
 ---
 # result definition
+
+# ok docs
 bool ok
 ---
 # feedback definition
+
```

Comment:
If there is no space here, does the comment apply the struct definition or the first member?
My guess is that it all is considered part of the struct definition and there is no comment for the first member. It might be good to test this case.

Reply:
> My guess is that it all is considered part of the struct definition and there is no comment for the first member. It might be good to test this case.

Yes, it works like that

Reply:
Extended the test case in https://github.com/ros2/rosidl/pull/632/commits/02ee80014cbbe28a648cc0b6891392f176e30e7f.
It was also being tested here https://github.com/ros2/rosidl/blob/5b42eb0a7041feea4e4be6e0ed5f5d0b56c17523/rosidl_adapter/test/data/action/Test.action#L1-L2.